src/components/Arrow/ArrowParts.module.scss

```
@@ -0,0 +1,36 @@
+.head {
+  min-width: 28px;
+  max-width: 28px;
+  height: 56px;
+  fill: var(--theme-color-1);
+}
```

Would it make sense if we created a variable for the size?

```suggestion
@use "sass:math";

$size: 56px;

.head {
  min-width: math.div($size, 2);
  max-width: math.div($size, 2);
  height: $size;
  fill: var(--theme-color-1);
}
```